api/GPUCommandEncoder.json

```diff
@@ -829,14 +829,10 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/GPUCommandEncoder/writeTimestamp",
           "spec_url": "https://gpuweb.github.io/gpuweb/#dom-gpucommandencoder-writetimestamp",
-          "tags": [
-            "web-features:webgpu"
```

Comment:
Thank you! This clearly shows the benefit of maintaining the mapping with tags in BCD 👍